src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -295,12 +295,12 @@ <h3 class="panel-title">
           <cd-submit-button [form]="formDir"
                             type="button"
                             (submitAction)="submit()">
-            <span i18n>{mode, select, editing {Update} cloning {Clone} copying {Copy} other {Create}}</span> RBD
+            <span i18n>{mode, select, editing {{{ActionLabels.EDIT}}} cloning {{{ActionLabels.CLONE}}} copying {{{ActionLabels.COPY}}} other {{{ActionLabels.CREATE}}}}</span> image
```

Comment:
I think that change is not properly parsed by ng-xi18n (instead of the `enum` literal expansion it's hitting the symbol itself). An alternative could be to treat that string in the TS side, rather than in the template.

Lots of errors are printed like this:
```ts
# ng xi18n --i18n-format xlf --i18n-locale en-US --output-path locale --progress=false && ngx-extractor -i 'src/**/*.ts' -f xlf -o src/locale/messages.xlf -l en-US

SKIP: Unknown argument type: 'PropertyAccessExpression' NodeObject {
  pos: 4159,
  end: 4178,
  flags: 0,
  transformFlags: undefined,
  parent: undefined,
  kind: 189,
  expression:
   IdentifierObject {
     pos: 4159,
     end: 4171,
     flags: 0,
     parent: undefined,
     escapedText: 'ActionLabels' },
  name:
   IdentifierObject {
     pos: 4172,
     end: 4178,
     flags: 0,
     parent: undefined,
     escapedText: 'CREATE' },
  _children:
   [ IdentifierObject {
       pos: 4159,
       end: 4171,
       flags: 0,
       parent: undefined,
       escapedText: 'ActionLabels' },
     TokenObject { pos: 4171, end: 4172, flags: 0, parent: [Circular], kind: 24 },
     IdentifierObject {
       pos: 4172,
       end: 4178,
       flags: 0,
       parent: undefined,
       escapedText: 'CREATE' } ] }
```

Reply:
It seems it has nothing to do with this but with a limitation in `ngx-extractor` to deal with some AST nodes, specifically 
`PropertyAccessExpression`:

https://github.com/ngx-translate/i18n-polyfill/blob/1db63d046d5a356570a259874e2e08ca9a0e4b2e/lib/extractor/src/abstract-ast-parser.ts#L20-L42